src/scenarios/Directory.Build.props

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why does this file and the Directory.Build.targets exist?

Reply:
it's placed there to override the arcade settings